scripts/docker/alpine/Dockerfile

```diff
@@ -10,14 +10,16 @@ RUN apk add --no-cache \
   eudev-dev \
   linux-headers \
   perl \
-  rust
+  rust \
+  clang-dev \
+  llvm-dev
 
 WORKDIR /parity
 COPY . /parity
-RUN cargo build --release --target x86_64-alpine-linux-musl --verbose
+ENV C=clang CXX=clang++
+RUN cargo build --release --features final --target x86_64-alpine-linux-musl --verbose
 RUN strip target/x86_64-alpine-linux-musl/release/parity
```

Comment:
I think we should use a rustflags for stripping the binary as well https://github.com/OpenEthereum/open-ethereum/blob/7d54e9258dd04671e6204769e1f624b7dbf496d0/scripts/gitlab/build-linux.sh#L13